src/extension.ts

```diff
@@ -172,7 +173,9 @@ export function activate(context: vscode.ExtensionContext): void {
         }
 
         const createdApp = <IAzureNode<WebAppTreeItem>>await node.createChild(this);
-
+        const createdAppConfig: SiteConfigResource = await createdApp.treeItem.client.getSiteConfig();
+        this.properties.os = createdAppConfig.kind ? createdAppConfig.kind : 'undefined';
+        this.properties.runtime = createdAppConfig.linuxFxVersion ? createdAppConfig.linuxFxVersion : 'undefined';
```

Comment:
This doesn't cover telemetry for all the 'create' entry points. We also want to track telemetry if they create it as a part of deploy or other actions